src/Microsoft.PowerApps.TestEngine/TestEngineEventHandler.cs

```diff
@@ -22,8 +22,8 @@ public class TestEngineEventHandler : ITestEngineEvents
         public static string UserInputExceptionLoginCredentialMessage = "   Invalid login credential(s). For more details, check the logs.";
         public static string UserInputExceptionTestConfigMessage = "   Invalid test config. For more details, check the logs.";
         public static string UserInputExceptionYAMLFormatMessage = "   Invalid YAML format. For more details, check the logs.";
-
         public static string UserAppExceptionMessage = "   [Critical Error] Could not access PowerApps. For more details, check the logs.";
+        public static string UserInputExceptionInvalidOutputPathMessage = "Invalid output directory; does not exist. For more details, check the logs.";
```

Comment:
nit nit: reorder this above UserAppExceptionMessage

Reply:
That doesn't make sense alphabetically,